homeassistant/components/london_underground/__init__.py

```diff
@@ -1 +1,33 @@
 """The london_underground component."""
+
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN as DOMAIN
+from .coordinator import LondonTubeCoordinator, TubeData
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up London Underground from a config entry."""
+
+    session = async_get_clientsession(hass)
+    data = TubeData(session)
+    coordinator = LondonTubeCoordinator(hass, data)
```

Comment:
Also, the coordinator requires the config entry to be passed it, so we should also update that

Reply:
done